### PR TITLE
fix: template label and link

### PR DESCRIPTION
**Pull request by banusha2**
## Context

To fix template label and link.
## Objective
Screen Shot 2023-02-24 at 2 35 04 PM

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/build-banner/component.js [hunk 1]
```diff
@@ -83,20 +83,19 @@ export default Component.extend({
...
-        }
+        const usageTemplate = {
+          name: template.name,
+          namespace: template.namespace,
+          version: template.version
+        };
 
-        this.set('template', usageTemplate)
-      })
+        return this.set('template', usageTemplate);
```
Comment (jithine) [inline app/components/build-banner/component.js:95]: is this `set` needed here ?

Comment (adong) [reply app/components/build-banner/component.js:95]: I dont think this syntax is correct.

How about? 

```
this.set('template', usageTemplate);

return this.template;
```

File: app/components/build-banner/component.js [hunk 2]
```diff
@@ -83,20 +83,21 @@ export default Component.extend({
...
+
+        this.set('template', usageTemplate);
 
-        this.set('template', usageTemplate)
-      })
+        return this.template;
+      });
     },
-    set(_, value ) {
+    set(_, value) {
```
Comment (adong) [inline app/components/build-banner/component.js:100]: I don't think `set` method is needed.

Comment (adong) [reply app/components/build-banner/component.js:100]: Also, the variable is not the same, one is setting `_template` and one is setting `template`. So, I would say just delete `set()` method

File: app/components/build-banner/component.js [hunk 3]
```diff
@@ -83,20 +83,21 @@ export default Component.extend({
...
-          'namsespace': template.namespace,
-          'version': template.version
-        }
+        const usageTemplate = {
+          name: template.name,
+          namespace: template.namespace,
+          version: template.version
+        };
+
+        this.set('template', usageTemplate);
```
Comment (adong) [inline app/components/build-banner/component.js:95]: Variable `template` is a computed property, `template: computed(...`

and set on the same variable name inside its own function does not really do anything. 


So this whole thing can be simplified as 

```js
  template: computed('templateId', {
    get() {
      const templateId = this.get('templateId');

      const templateProxy = ObjectPromiseProxy.create({
        promise: resolve(
          this.shuttle.getTemplateDetails(templateId)
            .then(template => {
              const usageTemplate = {
                name: template.name,
                namespace: template.namespace,
                version: template.version
              };

              return usageTemplate;
            })
          )
      });
    }
  })
```

Comment (adong) [reply app/components/build-banner/component.js:95]: The issue here actually is that this function itself has no return value, i.e. `templateProxy` was not returned. 

It ought to be something like this:

```
  template: computed('templateId', {
    get() {
      const templateId = this.get('templateId');

      return ObjectPromiseProxy.create({
        promise: this.shuttle.getTemplateDetails(templateId)
          .then(template => {
            const usageTemplate = {
              name: template.name,
              namespace: template.namespace,
              version: template.version
            };

            return usageTemplate;
          })
      });
    }
  }),
```

**Comment by adong** [comment]
Just need to 
1) Revert the changes for `app/components/build-banner/template.hbs`, since https://github.com/screwdriver-cd/ui/pull/886 is merged
2) Update file `app/components/build-banner/component.js` per comment https://github.com/screwdriver-cd/ui/pull/885#discussion_r1120660966

**Review by adong — APPROVED** [review]

**Pull request merged**